src/media/types/clean-up-result.ts

```diff
@@ -0,0 +1,7 @@
+import { DeleteResult } from 'typeorm';
+import { BatchStorageOperationStatistics } from '../providers/types/storage-provider';
+
+export interface CleanUpResult {
+  database: DeleteResult;
+  storage: BatchStorageOperationStatistics;
```

Comment:
Maybe rename it to something like "storageStatistics" so it is clear what this is?

Reply:
I think it's more explicit this way, because it shows that its (a) batched operations and (b) statistics for its operations only. Otherwise, you could assume it contains things like "amount of images saved in storage".